src/hooks/use_listen_message.rs

```diff
@@ -210,39 +210,41 @@ pub fn use_listen_message(cx: &ScopeState) -> &UseListenMessagesState {
                         });
                     }
                     info!("after write");
-                    // let room_name = if let Some(name) = message_event.room.name() {
-                    //     name
-                    // } else {
-                    //     let mut name = String::from("Unknown name room");
-                    //     let me = client.whoami().await.unwrap();
-                    //     let users = message_event.room.members().await;
-
-                    //     if let Ok(members) = users {
-                    //         let member = members
-                    //             .into_iter()
-                    //             .find(|member| !member.user_id().eq(&me.user_id));
-
-                    //         if let Some(m) = member {
-                    //             let n = m.name();
-
-                    //             name = String::from(n);
-                    //         }
-                    //     }
-
-                    //     name
-                    // };
-
-                    // handle_notification(
-                    //     NotificationItem {
-                    //         title: String::from(room_name),
-                    //         body: String::from(plain_message),
-                    //         show: true,
-                    //         handle: NotificationHandle {
-                    //             value: NotificationType::Click,
-                    //         },
-                    //     },
-                    //     notification.to_owned(),
-                    // );
+
+                    let room_name = match message_event.room.name() {
```

Comment:
Since you are returning name when it's `Some(_)` better use `unwrap_or_else` that would be more idiomatic.